src/index.js

```diff
@@ -148,10 +149,17 @@ if ( !! window.wcCalypsoBridge.isEcommercePlanTrial ) {
 		render: WoocommercePaymentsTaskPage,
 	} );
 
-	registerPlugin( 'wc-calypso-bridge-homescreen-slotfill-banner', {
-		render: CalypsoBridgeHomescreenBanner,
-		scope: 'woocommerce-admin',
-	} );
+	if ( window.wcCalypsoBridge.introductoryOffer ) {
+		registerPlugin( 'wc-calypso-bridge-homescreen-slotfill-banner', {
+			render: CalypsoBridgeIntroductoryOfferBanner,
```

Comment:
Since this component is not loaded for other than trial plans, let's make this a lazy import

Reply:
Correct me if I'm wrong, but it seems like RegisterPlugin doesn't currently support lazy-loaded React components. I've tried it, but nothing gets loaded. It's probably not using the Suspense component.

Do you happen to know if we have an example of a lazy-loaded plugin?

Reply:
Good question, I'm not sure if this works differently, but [PaymentGatewaySuggestions](https://github.com/Automattic/wc-calypso-bridge/blob/0233630e1a2a406600cfd6a3c1d0be81d3637a2c/src/index.js#L123) is lazy loaded and it's under a `registerPlugin` call but it's wrapped inside another component.

Perhaps we can try to wrap the lazy loaded component inside an empty component? If it doesn't work, then I think we can just skip this suggestion.